halibot/halibot.py

```diff
@@ -72,6 +73,7 @@ def add_instance(self, name, inst):
 	def _load_config(self):
 		with open("config.json","r") as f:
 			self.config = json.loads(f.read())
+			halibot.packages.__path__ = self.config.get("package-path", [])
```

Comment:
To make sure I'm understanding this, this won't change what needs to go in `package-path`, right? Those semantics should be the same?

Reply:
Correct, the `package-path` remains unchanged from previous versions.

Reply:
Good good, was afraid we'd have to change defaults again